package.json

```diff
@@ -21,23 +21,23 @@
     "chai": "^4.3.4",
     "chai-as-promised": "^7.1.1",
     "chrome-extension-cli": "^0.2.3",
-    "copy-webpack-plugin": "^6.4.1",
-    "css-loader": "^4.3.0",
+    "copy-webpack-plugin": "^10.1.0",
```

Comment:
Oh awesome! Glad you went through and grabbed the latest dependencies!

Reply:
Mostly a requirement. I played this game called "update one thing and figure out what else broke". I stopped when stuff went green.